docs/FullGuide.md

```diff
@@ -213,6 +271,9 @@ and
 
 For a blocking read or write that transfers the specified number of frames, set timeoutNanos greater than zero. For a non-blocking call, set timeoutNanos to zero. In this case the result is the actual number of frames transferred.
 
+Writing directly to a low latency stream is not supported on legacy. For best
```

Comment:
This is the first instance of "legacy" - do you mean "when the underlying API is OpenSLES"?

Reply:
Failure to set a callback may result in a non-low latency stream